lib/lurch/inflector.rb

```diff
@@ -1,30 +1,31 @@
 # frozen_string_literal: true
 
+require "dry/inflector"
+
 module Lurch
   class Inflector
     def initialize(inflection_mode, types_mode)
+      @inflector = Dry::Inflector.new do |inflections|
+        inflections.plural "people", "people"
```

Comment:
Why is this configuration required here?  Would we need a way to expose custom pluralization rules to the end-user?

Reply:
This was basically added only to make the tests run. Apparently dry-inflector does pluralize a little different than Inflecto. 
Yes, customizing the Inflector seems to be necessary.
However, I think we should also reduce the number of calls to a minimum, because this seems to be a big performance hit (https://github.com/peek-travel/lurch/issues/37) or remove the inflector completely. Both seems to be out of scope for this PR.